viz_scripts/generic_metrics.ipynb

```diff
@@ -145,15 +147,21 @@
     "file_name= 'ntrips_commute_mode_confirm%s' % file_suffix\n",
     "\n",
     "try:\n",
-    "    labels_mc = expanded_ct.query(\"Trip_purpose == 'Work'\").Mode_confirm.value_counts(dropna=True).keys().tolist()\n",
-    "    values_mc = expanded_ct.query(\"Trip_purpose == 'Work'\").Mode_confirm.value_counts(dropna=True).tolist()\n",
-    "    commute_quality_text = scaffolding.get_quality_text(expanded_ct, expanded_ct.query(\"Trip_purpose == 'Work'\"), \"commute\", include_test_users)\n",
+    "    if (len(dynamic_labels)):\n",
+    "        purpose_map_label =  scaffolding.mapping_labels(dynamic_labels, \"PURPOSE\")\n",
+    "        translation_work = purpose_map_label['work']\n",
+    "        trip_purpose_query = f\"Trip_purpose == '{translation_work}'\"\n",
+    "    else:\n",
+    "        trip_purpose_query = \"Trip_purpose == 'Work'\"\n",
+    "\n",
```

Comment:
I might let this slide now but this is super ugly code. Why not just use the raw `purpose_confirm` instead of the mapped value? That is guaranteed to be the same across all translations.